pic-build

```diff
@@ -21,6 +21,7 @@
 
 this_dir=`dirname $0`
 build_dir=".build"
+this_dir=$(cd $this_dir && pwd)
```

Comment:
this is too verbose with double-assign and invalid-in-between and also affects more tools. I will force-push updates to this PR.